modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -305,17 +317,16 @@ def _handle_triggers_within_batch(self, batch: list[tuple[int, int, int]], trigg
                 )
             else:
                 logger.info(f"Skipping training on empty trigger {trigger_id}]")
-            self._persist_pipeline_log()
```

Comment:
why do we remove this?
I'd like to keep it because this position is immediately after a trigger training / evaluation. We can persist the training / evaluation data so long as we have it.